scripts/build/termux_step_setup_variables.sh

```diff
@@ -76,6 +76,9 @@ termux_step_setup_variables() {
 	export prefix=${TERMUX_PREFIX}
 	export PREFIX=${TERMUX_PREFIX}
 
+	# Explicitly export in case the default was set.
+	export TERMUX_ARCH=${TERMUX_ARCH}
```

Comment:
I noticed this wasn't exported when cross-compiling and an arch wasn't passed in, ie the default arch was used, so I made sure it's always exported here, else `buildorder.py` couldn't find it.